target/linux/mvebu/base-files/etc/init.d/ethtool_gro

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2015-2016 OpenWrt.org
+# Copyright (C) 2016 LEDE-Project.org
+
+START=97
+boot() {
+which ethtool >/dev/null 2>&1 || exit 0
```

Comment:
This should be fixed in a different way. Turning it off for all board and for all protos looks rather like a hammer than a proper fix.

Reply:
Yes, obviously, the driver bug should be fixed. But until someone actually gets this done, this works fine as a workaround.

I can limit it to certain boards, but I've seen the same problem on my linksys ac1900 before, so don't really think it is omnia-specific.

I can add back the board-specific test if you prefer, or I can drop this patch entirely and just leave 6in4 broken; but I am not going to have time to debug this properly in the scope of this pull request. So which would you prefer? :)

Reply:
> So which would you prefer? :)

drop this patch entirely and just eave 6in4 broken :-)